buildkite/buildkite.go

```diff
@@ -57,6 +57,7 @@ type Client struct {
 	User              *UserService
 	Teams             *TeamsService
 	Tests             *TestsService
+	TestPlans					*TestPlansService
```

Comment:
[The nail that sticks out gets hammered down](https://en.wiktionary.org/wiki/the_nail_that_sticks_out_gets_hammered_down)